gwt-vuelidate/sources/src/main/java/nl/aerius/vuelidate/Validations.java

```diff
@@ -37,7 +37,7 @@ public class Validations implements JsPropertyMap<Object> {
   @JsProperty(name = "$anyError") public boolean anyError;
   @JsProperty(name = "$pending") public boolean pending;
 
-  @JsProperty(name = "$params") public JsObject params;
+  @JsProperty(name = "$params") public ValidationParams params;
```

Comment:
Does this count as 'Fully backward compatible'? ;)

Reply:
Yesn't

Sort of

Strictly speaking no

But yes

Params can still be cast to a JsObject, and, more importantly, it was never used